aspnetcore/tutorials/first-mongo-app.md

```diff
@@ -263,69 +263,263 @@ The preceding web API controller:
+1. Replace the Book comment with a book object as the JSON request body:
 
-   :::code language="csharp" source="first-mongo-app/samples/9.x/BookStoreApi/Program.cs" id="snippet_AddControllers" highlight="10-11":::
+   ```
+   POST {{BookStoreApi_HostAddress}}/api/books
+   Content-Type: application/json
 
-   With the preceding change, property names in the web API's serialized JSON response match their corresponding property names in the CLR object type. For example, the `Book` class's `Author` property serializes as `Author` instead of `author`.
+    {
+      "Name": "The Pragmatic Programmer",
+      "Price": 49.99,
+      "Category": "Computers",
+      "Author": "Andy Hunt"
+    }
+   
+   ###
+   ```
 
-1. In `Models/Book.cs`, annotate the `BookName` property with the [`[JsonPropertyName]`](xref:System.Text.Json.Serialization.JsonPropertyNameAttribute) attribute:
+1. Select the **Send request** link above the `POST` request line.
 
-   :::code language="csharp" source="first-mongo-app/samples/9.x/BookStoreApi/Models/Book.cs" id="snippet_BookName" highlight="2":::
+   The POST request is sent to the app, and the response is displayed in the **Response** pane. The response should include the newly created book with its assigned ID.
 
-   The `[JsonPropertyName]` attribute's value of `Name` represents the property name in the web API's serialized JSON response.
+1. Lastly, to delete a book, right-click the `/api/books/{id}, params (string id)` **DELETE** endpoint and select **Generate request**.
 
-1. Add the following code to the top of `Models/Book.cs` to resolve the `[JsonProperty]` attribute reference:
+   The following content is appended to the `BookStoreApi.http` file:
 
-   :::code language="csharp" source="first-mongo-app/samples/9.x/BookStoreApi/Models/Book.cs" id="snippet_UsingSystemTextJsonSerialization":::
+   ```
+   DELETE {{BookStoreApi_HostAddress}}/api/Books/{{id}}
+    
+   ###
+   ```
 
-1. Repeat the steps defined in the [Test the web API](#test-the-web-api) section. Notice the difference in JSON property names.
+1. Replace the `id` variable with one of the IDs returned from the earlier request, and click **Send request**. For example:
+
+   ```
+   DELETE {{BookStoreApi_HostAddress}}/api/Books/67f417517ce1b36aeab71236
+
+   ###
+   ```
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+This tutorial uses the [OpenAPI specification (openapi.json) and Swagger UI](xref:tutorials/web-api-help-pages-using-swagger) to test the API.
+
+1. Install Swagger UI by running the following command:
+
+  ```dotnetcli
+  dotnet add package NSwag.AspNetCore
+  ```
+
+The previous command adds the [NSwag.AspNetCore](https://www.nuget.org/packages/NSwag.AspNetCore/) package, which contains tools to generate Swagger documents and UI.
+Because our project is using OpenAPI, we only use the NSwag package to generate the Swagger UI.
+
+1. Configure Swagger middleware
```

Comment:
This is copy pasted from ["Tutorial: Create a controller-based web API with ASP.NET Core"](https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-9.0&tabs=visual-studio-code#configure-swagger-middleware)